### PR TITLE
Fixed #32863 -- Skipped system check for specifying type of auto-created primary keys on models with invalid app_label.

### PR DESCRIPTION
**Pull request by hramezani**
https://code.djangoproject.com/ticket/32863

### CODE REVIEW

File: django/db/models/base.py [hunk 1]
```diff
@@ -1310,16 +1310,23 @@ def _check_default_pk(cls):
                 cls._meta.pk.remote_field.parent_link
             ) and
             not settings.is_overridden('DEFAULT_AUTO_FIELD') and
-            not cls._meta.app_config._is_default_auto_field_overridden
+            (
+                not cls._meta.app_config or
+                not cls._meta.app_config._is_default_auto_field_overridden
+            )
```
Comment (felixxm) [inline django/db/models/base.py:1316]: I think we should ignore models with an invalid app label. I don't see much value in raising a warning for them.
```suggestion
            cls._meta.app_config and
            not cls._meta.app_config._is_default_auto_field_overridden
```

**Review by felixxm — COMMENTED** [review]
@hramezani Thanks for this patch :+1: Please add release notes, e.g.:
```
* Fixed a bug in Django 3.2 where a system check would crash on a model with an
  invalid ``app_label`` (:ticket:`32863`).
```

**Review by felixxm — COMMENTED** [review]
@hramezani Thanks for updates :+1: I moved release notes to the `3.2.5`.

**Comment by hramezani** [comment]
> @hramezani Thanks for updates  I moved release notes to the `3.2.5`.

Ah sorry for that and Thanks for taking care!

**Pull request merged**